src/util/defaults.ml

```diff
@@ -154,7 +154,8 @@ let _ = ()
       ; reg Experimental "exp.addr-context"      "false" "Ignore non-address values in function contexts."
       ; reg Experimental "exp.no-int-context"    "false" "Ignore all integer values in function contexts."
       ; reg Experimental "exp.no-interval32-context" "false" "Ignore integer values of the Interval32 domain in function contexts."
-      ; reg Experimental "exp.malloc-fail"       "false" "Consider the case where malloc fails."
+      ; reg Experimental "exp.malloc.fail"       "false" "Consider the case where malloc fails."
+      ; reg Experimental "exp.malloc.wrappers" "['myalloc','myalloc2','ldv_malloc','smalloc','no_malloc']"  "Loads a list of known malloc wrapper functions."
```

Comment:
I would argue that we should have all of the things we map to `malloc` or `calloc` in https://github.com/goblint/analyzer/blob/master/src/analyses/libraryFunctions.ml in this list by default.

If we have any of these functions appears in a benchmark and we have a body for them (and don't use the `special` edge therefore), we probably want to treat them as wrappers around malloc.

Reply:
Seems reasonable indeed. It would then also cover the original intent of forcefully making `kzalloc` and `kmalloc` special just to keep their locations but without being unsound.